.github/workflows/test.yml

```diff
@@ -73,10 +73,10 @@ jobs:
         run: |
           go vet -tags=example -v ./...
 
-      - name: go vet (atomic align)
+      - name: go vet (atomic align, error check)
         run: |
-          go install .github/workflows/vettools/atomicalign.go
-          go vet -vettool=$(which atomicalign)${{ startsWith(matrix.os, 'windows-') && '.exe' || '' }} -tags=example -v ./...
+          go install .github/workflows/vettools/vettools.go
+          go vet -vettool=$(which vettools)${{ startsWith(matrix.os, 'windows-') && '.exe' || '' }} -tags=example -v -errcheck.exclude=$PWD/.github/workflows/vettools/errcheck_excludes.txt ./...
```

Comment:
Do we need $PWD here?

Reply:
I believe so, because `go vet` runs `vettools` from different directories, so it needs an absolute path to the excludes file.